config.sample.yaml

```diff
@@ -111,7 +111,6 @@ services:
 
   - ID: "slackapi_service"
     Type: "slackapi"
-    UserID: "@slackapi:localhost" # requires a Syncing client
```

Comment:
I think you're confused. You still need a `UserID`, else who will send the messages as they arrive from the webhook? It just doesn't need to hit `/sync` to receive `!commands`. You just needed to remove the comment.